video.js

```diff
@@ -0,0 +1,90 @@
+import { onBindFirstLoad } from '../utils/jQueryBindFirst';
+import jQueryAjaxStop from '../utils/jQueryAjaxStop';
+import findElements from '../utils/findElements';
+import debounce from '../utils/debounce';
+
+const videoratio = 0.5625;
+
+export function setBackgroundImage(element, imageUrl) {
+  // Don't simply set "background:url(...)..." because this prop would override
+  // custom styling such as "background-size: cover".
+  element.setAttribute('style', `background-image:url(${imageUrl});background-color:#000;background-position:center center;background-repeat:no-repeat;`);
+}
+
+const debouncedResize = debounce(() => {
+  findElements('.container-lazyload').forEach((domContainerItem) => {
+    /* change ratio */
```

Comment:
Having to write `/* change ratio */` shouldn't be necessary. This is an indicator that the code needs to be refactored. See my other comments for more concrete suggestions.

Reply:
I inserted the comment for you to easier identify my changes, which isn't really necessary in a git, I suppose.